src/encoded/vis_indexer.py

```diff
@@ -46,7 +47,9 @@ def includeme(config):
     config.add_route('_visindexer_state', '/_visindexer_state')
     config.scan(__name__)
     registry = config.registry
-    registry['vis'+INDEXER] = VisIndexer(registry)
+    is_vis_indexer = registry.settings.get('visindexer')
```

Comment:
There are a number of processes running, this change only create a vis indexer for the vis indexer process.